packages/govern-create/utils/network.ts

```diff
@@ -0,0 +1,43 @@
+import 'dotenv/config'
+export function node_url(networkName: string): string {
```

Comment:
Can you elaborate the logic of this closer to me? not directly clear to me with just reading the code.